crashtracker-ffi/src/crash_info/stacktrace.rs

```diff
@@ -3,19 +3,26 @@
 
 use ::function_name::named;
 use datadog_crashtracker::{StackFrame, StackTrace};
-use ddcommon_ffi::{wrap_with_void_ffi_result, Handle, Result, ToInner, VoidResult};
+use ddcommon_ffi::{wrap_with_void_ffi_result, Error, Handle, ToInner, VoidResult};
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 //                                              FFI API                                           //
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#[allow(dead_code)]
+#[repr(C)]
+pub enum StackTraceNewResult {
```

Comment:
you can use `ddcommon_ffi::Result` e.g. `ddcommon_ffi::Result<Handle<StackTrace>>`

Reply:
If I use `ddcommon_ffi::Result<Handle<StackFrame>>` instead of `StackFrameNewResult`
The enum item name is 
`DDOG_CRASHT_STACK_FRAME_NEW_RESULT_OK_HANDLE_STACK_FRAME` (with renaming)
or
`DDOG_CRASHT_RESULT_HANDLE_FRAME_OK_HANDLE_STACK_FRAME`

If I use the new enum `StackFrameNewResult`, I got this
`DDOG_CRASHT_STACK_FRAME_NEW_RESULT_OK` which is shorter

Reply:
If you know a way to prevent a new enum to have shorter name, I'll be happy :)